packages/core/src/VNodes/ListNode.ts

```diff
@@ -1,4 +1,5 @@
 import { VElement } from './VElement';
+import { VNode } from './VNode';
```

Comment:
Same here, if you have anything particular to change in the commit message, given that you authored this version, please proceed :-)
Also, obviously, put yourself as author...

Reply:
Naive rebasing... Sorry!